syntax/std_os_wasm.go

```diff
@@ -25,3 +25,7 @@ func pathListSeparator() rel.Value {
 func cwd() rel.Value {
 	return rel.NewSet()
 }
+
+func file(rel.Value) rel.Value {
+	return rel.NewBytes([]byte{})
```

Comment:
`panic("not implemented")`

Consider doing the same for the existing functions in this file. `cwd` definitely should panic. While you're at it, replace `rel.NewSet()` with `rel.None`.